MessageRecovery/errors.rst

```diff
@@ -15929,7 +15929,7 @@ ZLINKFILE
 
 ZLINKFILE, Error while ZLINKing "xxxx"
 
-Run Time Error: This indicates that ZLINK command failed while trying to include routine xxxx in the image.
+Run Time Error: This indicates that ZLINK command failed while trying to include routine xxxx in the image. A jobbed off process will encounter a ZLINKFILE error if $ZROUTINES and $gtmroutines differ, and if it can be found only through $ZROUTINES, since it only inherits $gtmroutines.
```

Comment:
A jobbed off process ... -> Note that a jobbed off process could encounter a ZLINKFILE error if the $ZROUTINES ISV and $gtmroutines  environment variable differ, and the routine xxxx can only be found through $ZROUTINES but not through $gtmroutines (since the jobbed off process only inherits $gtmroutines).